runtime/kusama/src/xcm_config.rs

```diff
@@ -401,6 +416,11 @@ impl pallet_xcm::Config for Runtime {
 	// transfer.
 	type XcmReserveTransferFilter = Everything;
 	type Weigher = FixedWeightBounds<BaseXcmWeight, RuntimeCall, MaxInstructions>;
+	type DestinationWeigher = UniversalWeigherAdapter<
+		// use local weight for remote message and hope for the best.
+		FixedWeightBounds<BaseXcmWeight, (), MaxInstructions>,
```

Comment:
Since we really want to use the same weigher here, we should put the weigher into its own type alias and use it in both places